### PR TITLE
[Images] Refine text and created pages for Serve Images

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-4398

### CODE REVIEW

File: content/images/cloudflare-images/serve-images/adaptive-images-format.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: Adaptive image format
+meta:
+    title: Adaptive optimized format delivery
+layout: single
+weight: 1
+---
+
+# Adaptive optimized format delivery
+
+Cloudflare Images automatically serves AVIF and WebP when the browser supports these efficient image formats, regardless of the format you upload.
```
Comment (deadlypants1973) [inline content/images/cloudflare-images/serve-images/adaptive-images-format.md:12]: Is the word 'efficient' necessary here? or is it just acting as an adjective?

File: content/images/cloudflare-images/serve-images/serve-images-custom-domains.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+pcx_content_type: reference
+title: Custom domains
+layout: single
+meta:
+    title: Serve images from custom domains
+weight: 2
+---
+
```
Comment (deadlypants1973) [inline content/images/cloudflare-images/serve-images/serve-images-custom-domains.md:9]:
```suggestion
```

File: content/images/cloudflare-images/serve-images/serve-private-images-using-signed-url-tokens.md [hunk 3]
```diff
@@ -1,62 +1,67 @@
...
+layout: single
+meta:
+  title: Serve private images using signed URL tokens
+weight: 3
 ---
 
 # Serve private images using signed URL tokens
 
-If an image is marked to require a signed URL, it cannot be accessed without a token _unless_ it is being requested for a Variant that is set to always allow public access.
+If an image is marked to require a signed URL, it cannot be accessed without a token _unless_ it is being requested for a variant that is set to always allow public access.
```
Comment (deadlypants1973) [inline content/images/cloudflare-images/serve-images/serve-private-images-using-signed-url-tokens.md]:
```suggestion
If an image is marked to require a signed URL, it cannot be accessed without a token unless it is being requested for a variant that is set to always allow public access.
```

File: content/images/cloudflare-images/serve-images/serve-private-images-using-signed-url-tokens.md [hunk 4]
```diff
@@ -1,62 +1,67 @@
...
-    const imageDeliveryURL = new URL(
-      url.pathname.slice(1).replace('https:/imagedelivery.net', 'https://imagedelivery.net')
-    );
-    event.respondWith(generateSignedUrl(imageDeliveryURL));
-  });
-  ```
\ No newline at end of file
+3. Copy your key and use it to generate an expiring tokenized URL. 
+
+Here is an example Worker script that takes in a regular URL without a signed token and returns a tokenized URL that expires after one day. You can, however, set this expiration period to whatever you need, by changing the `const EXPIRATION` value.
```
Comment (deadlypants1973) [inline content/images/cloudflare-images/serve-images/serve-private-images-using-signed-url-tokens.md]:
```suggestion
Here is an example Worker that takes in a regular URL without a signed token and returns a tokenized URL that expires after one day. You can, however, set this expiration period to whatever you need, by changing the `const EXPIRATION` value.
```

Comment (deadlypants1973) [reply content/images/cloudflare-images/serve-images/serve-private-images-using-signed-url-tokens.md]: We are trying not to use the 'script' word anymore when talking about Workers.

File: content/images/cloudflare-images/tutorials/integrate-cloudflare-images.md [hunk 5]
```diff
@@ -210,7 +210,7 @@ Note as well that the displayed size now matches the real size of the image much
 
 ### Adaptive Optimized Format Delivery
 
-Another great built-in feature of Cloudflare Images is [Adaptive Optimized Format Delivery](/images/cloudflare-images/serve-images/#adaptive-optimized-format-delivery). This feature allows Cloudflare Images to automatically serve the best image format to the browser.
+Another great built-in feature of Cloudflare Images is [Adaptive Optimized Format Delivery](/images/cloudflare-images/serve-images/adaptive-images-format/). This feature allows Cloudflare Images to automatically serve the best image format to the browser.
```
Comment (deadlypants1973) [inline content/images/cloudflare-images/tutorials/integrate-cloudflare-images.md]:
```suggestion
[Adaptive Optimized Format Delivery](/images/cloudflare-images/serve-images/adaptive-images-format/) is a built-in feature of Cloudflare Images. This feature allows Cloudflare Images to automatically serve the best image format to the browser.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/images/cloudflare-images/serve-images/adaptive-images-format.md (hunk 1 above)
Comment (marciocloudflare) [reply content/images/cloudflare-images/serve-images/adaptive-images-format.md:12]: It's an adjective telling customers that AVIF and WebP are much more efficient than older formats, in that they occupy less space and have better quality

**Pull request merged**